src/library/scala/collection/convert/Accumulator.scala

```diff
@@ -72,6 +70,12 @@ abstract class Accumulator[@specialized(Double, Int, Long) A, +CC[X] <: mutable.
   }
 }
 
+trait AccumulatorOps[@specialized(Double, Int, Long) A, +CC[X] <: mutable.Seq[X], +C <: mutable.Seq[A]]
```

Comment:
Is this needed or just for regularity? I think it's not all that useful as it doesn't define any new operations.

Reply:
I think it is, but I can give it another shot.

Reply:
i managed to get rid of it.